src/Payments.Core/PaymentsContext.cs

```diff
@@ -8,8 +9,14 @@ public class PaymentsContext : DbContext
         public PaymentsContext(DbContextOptions<PaymentsContext> options) : base(options)
         {
             
+        }
+
+        [Obsolete("Just use for tests")]
+        public PaymentsContext()
+        {
+
         }
 
-        public DbSet<Invoice> Invoices { get; set; }
+        public virtual DbSet<Invoice> Invoices { get; set; }
```

Comment:
@jSylvestre that's strange, i've never seen it virtual in any of the example code or even in the EF tests themselves.  What error do you get when it isn't virtual?